consul-k8s.yaml

```diff
@@ -0,0 +1,61 @@
+package:
+  name: consul-k8s
+  version: 1.5.3
+  epoch: 0
+  description: The consul-k8s includes first-class integrations between Consul and Kubernetes.
+  copyright:
+    - license: MPL-2.0
+  dependencies:
+    runtime:
+      - go-discover
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/hashicorp/consul-k8s
+      expected-commit: 2cb07690ef4f557e65cd0606af30deafd18c3f6a
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      modroot: ./control-plane/cni
+      output: consul-cni
+      packages: .
+
+  - uses: go/build
+    with:
+      modroot: ./control-plane
+      output: consul-k8s-control-plane
+      packages: .
+
+subpackages:
+  - name: ${{package.name}}-cli
+    description: "CLI for interacting with Consul on Kubernetes"
+    pipeline:
+      - uses: go/build
+        with:
+          modroot: ./cli
+          output: consul-k8s
+          packages: .
+    test:
+      environment:
+        contents:
+          packages:
+            - go
```

Comment:
I think we should not to add `go` to runtime deps, is this required?

Reply:
Good catch. Removed.